### DISCUSSION TITLE
Osmand 4.8 severe issues with Android Auto

### DISCUSSION THREAD

**Discussion by DIRMAINT** [osmandapp/OsmAnd#20899] [General]
After the August update, Osmand has turned a very annoying piece of engineering work when using via Android Auto:

- Not starting
- Starting but turning non-responsive within a minute
- Turning non-responsive randomly
- Not getting a fix
- Loosing the fix
- Having the fix but map not scrolling
- Very slow initial screen render, and at the app resume, too (e.g. after a phone call): 30-60 seconds.

Usually, the execution resumes by pressing the locate button on the phone screen. Sometimes pressing once is ok, sometimes even five tries are needed. After the app gets stabilized, it may be running ok for several hundred kilometers. No matter whether the Osmand app is the foreground app on the phone or not.

What is interesting, turning back the car almost always leads to a non-responsive state or to loosing the fix. The same symptom can be seen often when the car is stationary for a few minutes. 

All other mapping applications (Google Maps, Tomtom, etc) are working without any problems.

The cache of the Osmand has been cleared (several times), as well as the cache of Android Auto.

Samsung A34, Android 14. 

Any hints?

**Comment by dmpr0** [comment]
Hi! Could you please send us the log file? You can find it by going to Menu > Help > Help Improve OsmAnd > Send Logcat Log.
If you prefer, you can attach the log file here or send it to support@osmand.net.

**Reply by zipav** [reply]
User sent log files:

**Reply by zipav** [reply]
[logcat (8).log](https://github.com/user-attachments/files/17458550/logcat.8.log)

[logcat (2) (1).log](https://github.com/user-attachments/files/17458549/logcat.2.1.log)

[logcat (1) (1).log](https://github.com/user-attachments/files/17458548/logcat.1.1.log)

**Comment by sjvudp** [comment]
Recently I'm also experiencing sudden app hangs that I cannot resolve while driving: Both, "wait" and "abort" are not resetting the app to a state where it responds.
And more seriously: While connected to the car, I cannot access the function to send logs, and when disconnected (and restarted?) the logs around the hang are gone.
As it is now, I cannot rely on the app to use as a navigation system in the car. Unfortunately.
